README.md

```diff
@@ -33,8 +33,8 @@ Add this component using HACS by searching for `Omnik Inverter Solar Sensor (No
 
 ## Manual installation
 
-Create a directory called `omnik_inverter` in the `<config directory>/custom_components/` directory on your Home Assistant instance.
-Install this component by copying all files in `/custom_components/omnik_inverter/` folder from this repo into the new `<config directory>/custom_components/omnik_inverter/` directory you just created.
+Create a directory called `omnik_inverter` in the `<config directory>/custom_components/` directory on your Home Assistant instance. In many installations, the <config directory> is where the configuration.yaml file is located. Create the directory custom_components first if it does not exist yet.
```

Comment:
You definitely don't want to have loose `<>` (HTML constructs) in a markdown file. Escape it with backticks, and do the same to the file and folder name.

Reply:
```suggestion
Create a directory called `omnik_inverter` in the `<config directory>/custom_components/` directory on your Home Assistant instance. In many installations, the `<config directory>` is where the `configuration.yaml` file is located. Create the directory `custom_components/` first if it does not exist yet.
```

Reply:
Though in hindsight this paragraph doesn't even tell users how to most conveniently download the necessary folder. Assuming most users install over the cmdline, we could even suggest a copy-pastable example like:

```bash
curl -L https://github.com/robbinjanssen/home-assistant-omnik-inverter/archive/refs/heads/main.zip | bsdtar xv --strip-components=1 home-assistant-omnik-inverter-main/custom_components/omnik_inverter/
```

This should create the `custom_components/omnik_inverter/` directory structure and only unzip the files within that folder in this repo to it.